src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -176,6 +180,24 @@ default boolean isEmpty() {
     byte[] getBases();
 
     /**
+     * @return The number of bases in the read sequence.
+     * This default implementation calls getBases().length
+     * Subclasses may override.
+     */
+    default int getBaseCount(){
+        return getBases().length;
```

Comment:
This may be a better name then `getLength` since getLength is easily confused with getting the aligned length.  We shouldn't have both though.